tdrs-backend/tdpservice/search_indexes/documents.py

```diff
@@ -23,54 +23,51 @@ class Django:
 
         model = T1
         fields = [
-            'record',
-            'rpt_month_year',
+            'record_type',
+            'reporting_month',
             'case_number',
-            'disposition',
-            'fips_code',
-
             'county_fips_code',
             'stratum',
             'zip_code',
             'funding_stream',
+            'disposition',
             'new_applicant',
-            'nbr_of_family_members',
+            'family_size',
```

Comment:
@andrew-jameson OFA will need a cross-walk of the old data elements columns to new ones. our existing syntax relies on the old ones.  cc: @lfrohlich @ttran-hub